src/containers/TemplateBuilder/Templates.tsx

```diff
@@ -30,11 +43,11 @@ const columns: Columns = [
 
   {
     title: '',
-    render: ({ template: { version, versionTimestamp } }) => (
+    render: ({ template }) => (
       <React.Fragment key="version">
-        <TextIO text={String(version)} title="version" maxLabelWidth={70} />
+        <TextIO text={getVersionString(template)} title="version" maxLabelWidth={70} />
         <TextIO
-          text={String(versionTimestamp.toFormat('dd MMM yy'))}
+          text={String(template.versionTimestamp.toLocaleString(DateTime.DATETIME_SHORT))}
```

Comment:
This is a more appropriate timestamp format -- since we can have multiple versions throughout the day, we need more than just DD/MM/YY.